egs/mobvoi/v1/local/chain/tuning/run_tdnn_1a.sh

```diff
@@ -0,0 +1,293 @@
+num_jobs_final=5
+chunk_width=140,100,160
+common_egs_dir=
+reporting_email=
+dim=80
+bn_dim=20
+frames_per_iter=3000000
+bs_scale=0.0
+train_set=train_shorter
+combined_train_set=train_shorter_sp_combined
+test_sets="dev eval"
+aug_prefix="rev1 noise music babble"
+export LC_ALL=en_US.UTF-8
+wake_word="嗨小问"
+export LC_ALL=C
+
+# End configuration section.
+echo "$0 $@"  # Print the command line for logging
+
+. ./cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+if ! cuda-compiled; then
+  cat <<EOF && exit 1
+This script is intended to be used with GPUs but you have not compiled Kaldi with CUDA
+If you want to use GPUs (and have them), go to src/, and configure and make on a machine
+where "nvcc" is installed.
+EOF
+fi
+
+gmm_dir=exp/${gmm}
+ali_dir=exp/${gmm}_ali_${train_set}_sp
+lat_dir=exp/chain/${gmm}_${train_set}_sp_lats
+combined_lat_dir=exp/chain/${gmm}_${combined_train_set}_lats
+combined_train_data_dir=data/${combined_train_set}_hires
+lores_train_data_dir=data/${train_set}_sp
+
+lang=data/lang_chain
+lang_decode=data/lang_chain_decode
+tree_dir=exp/chain/tree  # it's actually just a trivial tree (no tree building)
+dir=exp/chain/tdnn_${affix}
+
+for f in $combined_train_data_dir/feats.scp \
+  $lores_train_data_dir/feats.scp $gmm_dir/final.mdl $ali_dir/ali.1.gz; do
+  [ ! -f $f ] && echo "$0: expected file $f to exist" && exit 1
+done
+
+if [ $stage -le 1 ]; then
+  echo "$0: creating lang directory $lang with chain-type topology"
+  # Create a version of the lang/ directory that has one state per phone in the
+  # topo file. [note, it really has two states.. the first one is only repeated
+  # once, the second one has zero or more repeats.
+  if [ -d $lang ]; then
+    if [ $lang/L.fst -nt data/lang/L.fst ]; then
+      echo "$0: $lang already exists, not overwriting it; continuing"
+    else
+      echo "$0: $lang already exists and seems to be older than data/lang ..."
+      echo " ... not sure what to do.  Exiting."
+      exit 1;
+    fi
+  else
+    rm -rf $lang
+    cp -r data/lang $lang
+    silphonelist=$(cat $lang/phones/silence.csl) || exit 1;
+    nonsilphonelist=$(cat $lang/phones/nonsilence.csl) || exit 1;
+    # Use our special topology... note that later on may have to tune this
+    # topology.
+    local/gen_topo.pl 4 1 $nonsilphonelist $silphonelist >$lang/topo
+  fi
+fi
+
+if [ $stage -le 2 ]; then
+  # Get the alignments as lattices (gives the chain training more freedom)
+  # use the same num-jobs as the alignments
+  steps/align_fmllr_lats.sh --nj 75 --cmd "$train_cmd" ${lores_train_data_dir} \
+    data/lang $gmm_dir $lat_dir
+  rm $lat_dir/fsts.*.gz # save space
+fi
+
+if [ $stage -le 3 ]; then
+  local/copy_lat_dir.sh --nj 75 --utt-prefixes "$aug_prefix" \
+    $combined_train_data_dir $lat_dir $combined_lat_dir
+fi
+
+if [ $stage -le 4 ]; then
+  # Build a tree using our new topology.  We know we have alignments for the
+  # speed-perturbed data (local/nnet3/run_ivector_common.sh made them), so use
```

Comment:
The comment 

> local/nnet3/run_ivector_common.sh made them

is actually incorrect.

It is `steps/align_fmllr.sh` that made them.